test/relay_api_test.ts

```diff
@@ -0,0 +1,85 @@
+import * as fetchMock from 'fetch-mock';
+import * as chai from 'chai';
+import * as BigNumber from 'bignumber.js';
+import * as Sinon from 'sinon';
+import {ZeroEx} from '../src/';
+import {chaiSetup} from './utils/chai_setup';
+import {Web3Wrapper} from '../src/web3_wrapper';
+import {SchemaValidator, schemas} from '0x-json-schemas';
+
+chaiSetup.configure();
+chai.config.includeStack = true;
+const expect = chai.expect;
+
+// We need to have a mock API server in order to test that
+describe('Relay API', () => {
+    const relayUrl = 'https://example.com';
```

Comment:
What about `https://api.relayer.com`

Reply:
`https://api.relayer.com` seems confusing. It's not clear - if it's a real domain.
example.com is a standard way to express, that it doesn't matter and it's an example domain.
No one will expect to get real api responces from this one.

Reply:
come on...